multichain/openrpc.yaml

```diff
@@ -5,179 +5,223 @@ info:
   description: >-
     This provides the specs for the MultiChain API Layer for the MetaMask API
 methods:
-  - name: "provider_authorize"
-    paramStructure: "by-name"
+  - name: provider_request
+    paramStructure: by-name
     params:
-      - name: "requiredScopes"
-        description: "Scopes that the wallet must support in order to be used with this provider."
+      - name: scope
+        description: a valid `scopeObject` that has been previously authorized via `provider_authorize`
+        required: true
+        schema:
+          $ref: "#/components/schemas/Scope"
+      - name: request
+        description: an object containing a JSON-RPC request with `method` and `params`
+        deprecated: false
+        required: true
         schema:
           type: "object"
+          properties:
+            method:
+              type: "string"
+            params:
+              true
```

Comment:
just making sure this is the correct way to specify optional params

Reply:
its not optional but `true` here would just mean `any` if it was typescript